src/components/snackbar/styles.module.css

```diff
@@ -0,0 +1,20 @@
+.snackbarContainer{
+    position: fixed;
+    right:2rem;
+    bottom:2rem;
+    display:flex;
+    background-color: blue;
+    color: var(--text-light);
+    padding:0.75rem 1rem 0.5rem 1rem;
```

Comment:
padding: 0.75 1rem 0.5rem;

Reply:
it worked with 
padding: 0.75rem 1rem 0.5rem;